conf/local-config.json

```diff
@@ -12,5 +12,9 @@
   "partners_metadata_path": "/com.uid2.core/test/partners/metadata.json",
   "clients_metadata_path": "/com.uid2.core/test/clients/metadata.json",
   "operators_metadata_path": "/com.uid2.core/test/operators/metadata.json",
-  "optout_observe_only": false
+  "optout_observe_only": false,
+  "aws_kms_jwt_signing_public_keys": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA9tXqXNeeuA3lmfNF2AF06OLY6icxEOlXaZEXyh62ieFjakvDeR438s5gA1ToKqpW5uTPkKtP1YjHf16ofjlDYcLxCKOombjW1m2t/3FFfWRaZgr/SFr5TaI3N86tAvf313kql8GVrre/HQLhaHUXQJxDYh3/Wl7+KSMrMwMQCwkgnf1AmGUcPl1AzxuKUia8IM9joZKq9UpxRdWkLeqJ47d0bX5+SFLOwRhFxnejNx3iJYpEi43aFxLnr1WmTtMrIkHYmA2GENnhXsrV3n0bqLUah+HS6S+m3lBUW09TEFbagtktCtsFqvE+YExXo8b+hOnlakQtsbtJY0xg8Z1AtwIDAQAB,MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA9tXqXNeeuA3lmfNF2AF06OLY6icxEOlXaZEXyh62ieFjakvDeR438s5gA1ToKqpW5uTPkKtP1YjHf16ofjlDYcLxCKOombjW1m2t/3FFfWRaZgr/SFr5TaI3N86tAvf313kql8GVrre/HQLhaHUXQJxDYh3/Wl7+KSMrMwMQCwkgnf1AmGUcPl1AzxuKUia8IM9joZKq9UpxRdWkLeqJ47d0bX5+SFLOwRhFxnejNx3iJYpEi43aFxLnr1WmTtMrIkHYmA2GENnhXsrV3n0bqLUah+HS6S+m3lBUW09TEFbagtktCtsFqvE+YExXo8b+hOnlakQtsbtJY0xg8Z1AtwIDAQAB",
```

Comment:
I see it says public key, is it ok to be checked in? Or is this a fictitious test key?

Reply:
Public keys are not sensitive, so it can be checked in. It is a key we can use in e2e testing.